server/bin/www.js

```diff
@@ -75,6 +76,26 @@ function onListening() {
   debug("Listening on " + bind);
 }
 
+/**
+ * Database Connection
+ */
+
+const dbUsername = process.env.DEV_DB_USERNAME;
+const dbPassword = process.env.DEV_DB_PASSWORD;
+const connectionString = `mongodb+srv://${dbUsername}:${dbPassword}@dex-dev-jpy4j.mongodb.net/dex-dev?retryWrites=true&w=majority`;
+
+mongoose.connect(connectionString, { useNewUrlParser: true , useUnifiedTopology: true });
+mongoose.connection.on("error", (err) => {
+  console.error(err);
+  console.log("%s MongoDB connection error. Please make sure MongoDB is running.");
+  process.exit();
+});
+
+mongoose.connection.once("open", function() {
+    console.log("We connected");
+});
+
+
```

Comment:
Moved all mongoose connection code to `www.js` file in `bin/`. This has all the web server setup code